Solutions/LargestPrimeFactor.cs

```diff
@@ -0,0 +1,36 @@
+namespace Solutions;
+
+public class LargestPrimeFactor : ProblemSolverBase
+{
+  public LargestPrimeFactor()
+  {
+    ProblemID = 3;
+  }
+  public override void Solve()
+  {
+    Answer = GetLargestPrimeFactor(600851475143).ToString();
+  }
+
+  long GetLargestPrimeFactor(long number)
+  {
+    long largestPrime = 0;
+    long divisor = 2;
+
+    while (number != 1)
+    {
+      if (number % divisor == 0)
+      {
+        number = number / divisor;
+      }
+      else
+      {
+        divisor++;
+      }
+      if (divisor > largestPrime){
+        largestPrime = divisor;
+      }
+    }
+
+    return largestPrime;
+  }
+}
```

Comment:
Can you setup a linter as commit hooks? I don't know how there are done in c# but for python it'd be pre-commit and for js there is husky, I'm sure c# has something similar